server/claim/claim.service.ts

```diff
@@ -378,7 +388,9 @@ export class ClaimService {
                 );
 
             processedClaim.content =
-                processedClaim.contentModel === ContentModelEnum.Speech
+                processedClaim.contentModel === ContentModelEnum.Speech ||
```

Comment:
suggestion (non-blocking): this is getting increasingly complex, we should move to a function and use the strategy pattern.